modules/highlight.js

```diff
@@ -0,0 +1,423 @@
+class MarketHighlights {
+    static id = "market_highlights";
+    static displayName = "Market Highlights";
+    static icon = MarketHighlights.favoriteTemplate(); // TODO
+    static category = "economy";
+    css = `
+.tracker-highlight-button {
+    width: 33px;
+    height: 33px;
+    margin-left: 1px;
+    margin-right: 1px;
+    padding: 1px;
+    cursor: pointer;
+}
+
+.svg-inactive > * {
+    fill: none;
+}
+
+.favorite-highlight {
+    border: 3px solid white;
+}
+
+.heat-highlight {
+    border: 3px solid red;
+}
+
+/* Handling multiple highlights at once */
+.quantile-borders .favorite-highlight {
+    box-shadow: 0 0 0 3px white;
+}
+
+.quantile-borders .heat-highlight {
+    box-shadow: 0 0 0 3px red;
+}
+
+.favorite-highlight.heat-highlight {
+    border: 3px solid white;
+    box-shadow: 0 0 0 3px red;
+}
+
+.quantile-borders .favorite-highlight.heat-highlight {
+    box-shadow: 0 0 0 3px white, 0 0 0 6px red;
+}
```

Comment:
Sometimes spaces in between sometimes not

Reply:
Was correct, but I changed it such that all classes are directly set on the item node.